ansible/templates/usb/scripts/prompt-config.sh.j2

```diff
@@ -1347,6 +1362,18 @@ console=$(getanswer "config_console")
 # prompt:
 if [[ -z ${console} || $(tty) != "/dev/console" ]]; then
 	read continue;
+	# automatic installation is always advanced
+	printheader "Advanced installation"
+	printf "
+The advanced installation will allow you to customize additional network
+options, such as VLANs, multiple IP addresses, NTP servers, etc.
+Basic installation is suffcient for most installations\n\n"
+	promptbool "\nEnable advanced installation? (y/n)" "n"
```

Comment:
The question about advanced_install is displayed only `if [[ $(getanswer "skip_instructions") != "true" ]]`. I hope this should be a sufficient default.